patch/0011-arm64-dts-marvell-Add-Nokia-7215-IXS-A1-board.patch

```diff
@@ -137,6 +257,8 @@ index 49f274add..310b57e47 100644
  dtb-$(CONFIG_ARCH_MVEBU) += ac5-98dx35xx-rd.dtb
  dtb-$(CONFIG_ARCH_MVEBU) += armada-7020-comexpress.dtb
 +dtb-$(CONFIG_ARCH_MVEBU) += 7215-ixs-a1.dtb
+
+base-commit: 6b6f1082cb46d72823b7ea99c058c601668ba1d3
```

Comment:
Is this line meant to be part of the patch description, or the file changes? If it's part of the description, can this be moved to the top, below the existing description?"

Reply:
This is not part of the file changes. This is the upstream base commit ID intended for the upstream maintainers review. This is autogenerated when git-format patch executed on upstream repo.
This has no effect on SONiC.

Do you recommend removing this line from SONiC patch?